docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '2'
+services:
+  myapprest:
+    build: rest-api-security/
+    container_name: rest-api-security
+    ports:
+      - 8008:8008
+    restart: always
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/security_db
+      - SPRING_DATASOURCE_USERNAME=postgres
+      - SPRING_DATASOURCE_PASSWORD=postgre
+      - SERVER_PORT:8008
+    links:
+      - postgres
+    networks:
+      - mynetwork
+
+  myappweb:
+    build: web-api-security/
+    container_name: web-api-security
+    ports:
+      - 8080:8080
+    restart: always
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/security_db
```

Comment:
This localhost will reference to local container network, but Postgres resides inside different container.

To make it work you can access container by its name jdbc:postgresql://postgres:5432/security_db

Reply:
Done